src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-size: 30px;
+  padding: $square-paddings;
```

Comment:
The variable `$square-paddings` is used here. Make sure this variable is defined in your SCSS files, otherwise, it will cause an error during compilation.